common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1304,22 +1305,22 @@
     "figura.docs.text_task.set_alignment": "Sets this Text alignment\nCan be either \"LEFT\", \"RIGHT\" or \"CENTER\"\nDefault \"LEFT\"",
     "figura.docs.text_task.has_shadow": "Checks if this task text has shadow",
     "figura.docs.text_task.set_shadow": "Toggles if the Text should render with a drop shadow\nNot compatible with \"Outline\" mode",
-    "figura.docs.text_task.has_outline": "Checks if this task text has outline",
-    "figura.docs.text_task.set_outline": "Toggles if the Text should render with a outline\nAlways renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
+    "figura.docs.text_task.has_outline": "Checks if this task text has an outline",
+    "figura.docs.text_task.set_outline": "Toggles, if the text should render with an outline\nAlways, renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
```

Not sure why that first comma is there.
Changed the second line to hopefully be a bit more clear.
```suggestion
    "figura.docs.text_task.set_outline": "Toggles if the text should render with an outline\nThe text will always render at full brightness if it is given an outline\nNot compatible with \"Shadow\" and \"Emissive\" modes",
```